inference/inference.py

```diff
@@ -337,7 +336,7 @@ def autoregressive_inference(params, ic, valid_data_full, model):
             for k in range(len(out_names)):
               name = f'{metric_names[i]}_{time_name}/ic{ic}/channel{k}-{out_names[k]}'
               try:
-                wandb.log({name: all_metrics[i, j, k]})
+                wandb.log({name: all_metrics[i, j, k], 'epoch': params.epoch}, step=params.iters)
```

Comment:
We could avoid having to specify step or add new 'epoch' metric if we log these to wandb outside of this function (see other comment).

Reply:
right